components/cli/kubernetes/kubecli.go

```diff
@@ -37,6 +37,8 @@ type KubeCli interface {
 	GetInstancesNames() ([]string, error)
 	GetCell(cellName string) (Cell, error)
 	GetComposite(compositeName string) (Composite, error)
+	GetInstanceBytes(instanceKind, InstanceName string) ([]byte, error)
+	Describe(cellName string) error
```

Comment:
Is this method only for a cell or both for cells and composites? Rename the method name and parameter name accordingly.

Reply:
Currently "cellery describe" command is used for describing cell images and cell instances. Therefore Describe interface function is only used for cells. Updated method name to "DescribeCell" as requested.

Modify "cellery describe" command to describe composite images, composite instances as well?

Reply:
No lets keep it as it is for now.